cmd/podman/validate/args.go

```diff
@@ -98,6 +102,9 @@ func CheckAllLatestAndCIDFile(c *cobra.Command, args []string, ignoreArgLen bool
 // CheckAllLatestAndPodIDFile checks that --all and --latest are used correctly.
 // If withIDFile is set, also check for the --pod-id-file flag.
 func CheckAllLatestAndPodIDFile(c *cobra.Command, args []string, ignoreArgLen bool, withIDFile bool) error {
+	if registry.IsRemote() {
```

Comment:
I think you still want to do this check to make sure the user did not specify --all and --podidfile foobar.

Reply:
the current logic does not indicate that does it? i looked at this a couple of times.

Reply:
I think we still need it as we also check for `args`.

Reply:
I think he's right, this function does more than just check latest so we can't completely bypass